City-of-Pori/pori-events
# PORI-229: Event page theming

Event page theming. FontAwesome icons are not working for some reason.

To test:
1. drush cim -y && drush updb -y && drush cr
2. go and create an event. Provide as much info as you can and add multiple images to get carousel working. (If you get "File could not be uploaded" when adding image, check the files directorys permissions)
3. after you've done the event, go to its page and confirm that the page has been styled.
